playwright.config.ts

```diff
@@ -13,32 +13,36 @@ import { devices } from '@playwright/test';
 const config: PlaywrightTestConfig = {
   testDir: './tests',
   /* Maximum time one test can run for. */
-  timeout: 30 * 1000,
+  timeout: 120 * 1000,
   expect: {
     /**
      * Maximum time expect() should wait for the condition to be met.
      * For example in `await expect(locator).toHaveText();`
      */
-    timeout: 10000,
+    timeout: 15000,
   },
-  /* Run tests in files in parallel */
-  fullyParallel: true,
+  /* We don't want to run tests in parallel because we have the same backend, tests may conflict with eachother */
+  fullyParallel: false,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Global setup file for initializing before all other tests */
   globalSetup: require.resolve('./tests/globalSetup.ts'),
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
```

Comment:
What is this retrying? Could this prevent heisenbugs from being detected in CI unless they fail 3 times in a row?

Reply:
Yes, it retries on CI to prevent heisenbugs/flakey tests. The Playwright Report that is still output includes information on flakey tests. We could reduce it to 1 retry only... I don't think any of the tests should be flakey anymore right now (after this change).